### ISSUE TITLE
0.1.8 introduces a breaking change

### ISSUE THREAD

**Issue by gisenberg** [labocho/thor-zsh_completion#7]
Hey folks,

It looks like today's release of 0.1.8 includes a breaking change. In previous versions of `thor-zsh_completion`, the following code worked:

```ruby
require 'thor'
require 'thor/zsh_completion'
```

As of 0.1.8, the require of `thor/zsh_completion` fails with a missing file.

**Comment by labocho** [comment]
Thank you for reporting the issue!

But I couldn't see the problem in my environment (no error occurred).
Please reinstall `thor-zsh_completion` gem and retry.
If the issue reproduces, please tell me the following information.

1. Ruby version (by `ruby -v`)
2. `thor` and `thor-zsh_completion` gem version (by `gem list thor`)
3. Error message and backtrace for `require 'thor/zsh_completion'`

**Comment by belden** [comment]
I'm experiencing this too, on a fresh Ubuntu 20.10 installation.

```
belden@lucifer:~$ ruby -v
ruby 2.7.1p83 (2020-03-31 revision a0c7c23c9c) [x86_64-linux-gnu]
belden@lucifer:~$ gem list thor

*** LOCAL GEMS ***

thor (0.20.3, 0.19.4)
thor-zsh_completion (0.1.8)
belden@lucifer:~$ ruby -e "require 'thor/zsh_completion'"
Traceback (most recent call last):
        2: from -e:1:in `<main>'
        1: from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require'
/usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:92:in `require': cannot load such file -- thor/zsh_completion (LoadError)
        3: from -e:1:in `<main>'
        2: from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:156:in `require'
        1: from /usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:168:in `rescue in require'
/usr/lib/ruby/2.7.0/rubygems/core_ext/kernel_require.rb:168:in `require': cannot load such file -- thor/zsh_completion (LoadError)
```

**Comment by labocho** [comment]
Thank you for the response!

I reproduced the problem in the following situation.

```
# in ubuntu/groovy64 vagrant box
sudo apt install ruby
sudo gem install thor-zsh_completion:0.1.8
ruby -e "require 'thor/zsh_completion'"
```

The problem is caused by the file mode of `lib/thor/zsh_completion.rb`. It is `600`, so you could not load the gem installed by another user.

This is a bug. I fixed it and released v0.1.9.
Please `gem install thor-zsh_completion:0.1.9` and retry it.

**Comment by belden** [comment]
Confirmed, works for me. Thanks!
